### PR TITLE
ISSD-8 [Learn] Add script on Learn Article Template to remove the slash at the end of the URL

### PR DESCRIPTION
**Pull request by MariargMorais**

### CODE REVIEW

**Comment by MariargMorais** [comment]
ci:test:sf

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-article-template/ddm-template.ftl [hunk 1]
```diff
@@ -1,5 +1,14 @@
 <#include "${templatesPath}/SVG">
 
+<script>
+	currentSite = window.location.href;
```
Comment (joseabelenda) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-article-template/ddm-template.ftl:4]: let href = window.location.href;

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-article-template/ddm-template.ftl [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+	
+	if (currentSite.endsWith("/")){
+		currentSite = currentSite.substring(0, currentSite.length - 1);
+		window.location.assign(currentSite);
+		}
```
Comment (joseabelenda) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-article-template/ddm-template.ftl:9]: Identation

**Review by joseabelenda — COMMENTED** [review]
Very nice!
Please see my comments.

**Pull request closed without merging**